modules/pipeline/providers/queuemanager/load.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queuemanager
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+
+	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/pkg/strutil"
+)
+
+// loadRunningPipelines load from two sources:
+// - db
+// - etcd
+func (q *provider) loadRunningPipelines(ctx context.Context) {
```

Comment:
please give a better func name, load is too common.

Reply:
adjust to `loadNeedHandledPipelinesWhenBecomeLeader`. you will like it.